public/lesswrong-editor/src/ckeditor5-math/mathui.js

```diff
@@ -0,0 +1,282 @@
+/* eslint-disable no-tabs */
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import ClickObserver from '@ckeditor/ckeditor5-engine/src/view/observer/clickobserver';
+import ContextualBalloon from '@ckeditor/ckeditor5-ui/src/panel/balloon/contextualballoon';
+import clickOutsideHandler from '@ckeditor/ckeditor5-ui/src/bindings/clickoutsidehandler';
+import uid from '@ckeditor/ckeditor5-utils/src/uid';
+import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import MainFormView from './ui/mainformview';
+
+// Need math commands from there
+import MathEditing from './mathediting';
+import { defaultConfig, getSelectedMathModelWidget, resizeInputElement } from './utils';
+
+import mathIcon from './math.svg';
+
+const mathKeystroke = 'Ctrl+4';
```

Comment:
I think this was previously "Ctrl+4 or Cmd+4, whichever"? It needs to support both because there are operating-system-specific hotkeys that this conflicts with; on MacOS, Ctrl+4 is okay but Cmd+4 will switch browser tab, whereas on Windows and Linux it's the reverse. Cmd/Ctrl+M has a similar problem.

Reply:
CKeditor automatically takes care of those differences.